xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1385,14 +1388,18 @@ void CAdvancedSettings::AddSettingsFile(const std::string &filename)
   m_settingsFiles.push_back(filename);
 }
 
-float CAdvancedSettings::GetLatencyTweak(float refreshrate)
+float CAdvancedSettings::GetLatencyTweak(float refreshrate, bool isHDREnabled)
 {
   float delay = m_videoDefaultLatency;
   for (int i = 0; i < (int) m_videoRefreshLatency.size(); i++)
   {
     RefreshVideoLatency& videolatency = m_videoRefreshLatency[i];
     if (refreshrate >= videolatency.refreshmin && refreshrate <= videolatency.refreshmax)
+    {
       delay = videolatency.delay;
+      if (isHDREnabled && videolatency.hdrlatency != 0.0f)
+        delay += videolatency.hdrlatency;
```

Comment:
I'm not sure about making the hdr latency a delta rather than an absolute value. Is there a reason you went that way?

Reply:
If you feel up to it, changing the loop style to `for (const auto& videoLatency : m_videoRefreshLatency)` would be nice.

Reply:
Good question, I didn't know wich to chose either. 
I figured it's always an "extra" delay and would get less confusing for people with negative value for "base" delay, for example. If you know the actual value of added latency for HDR mode (in given refresh range) for your display, it's a value you can input here and it will work. 

Note : I know the "delay" I input for Kody has nothing to do with my actual display latency. Example : in MPC-HC I used 250ms (or so) of audio delay and "30ms" in advanced setting in Kody to get the same result. I don't know why yet.

Reply:
Based on your experience, is the extra delay the same for all refresh rates? I have no data to contribute, seems my screen doesn't add any delay for hdr.
It would make sense to create a "global" hdr extra delay under the latency tag, similar to the existing "delay" setting, to be used as default for all refresh rate unless an override is defined for a specific refresh rate.

Reply:
Not tested but... I'm pretty sure it varies the same way as refresh rate specific 'delay' does. 

All (my) HDR content is between 23-25 fps. So Kodi switches the TV to 23-25hz, real cinema is on, all kind things adding latency. In my case I estimated the HDR extra cost to be around 70ms in this range. 
There is no way imo HDR costs you 70ms in gaming mode at 60hz or 120hz. 
(I don't know about any 60fps HDR video content, for now)

I suppose one could test : playback hdr video (23-24fps) at 60hz or more (with 23-25hz removed from whitelist) and verify it has lesser impact. 

On my monitor (240hz), I don't notice any delay.

Reply:
Although you're right for consistency there is no reason not to add a global (default) setting aswell. 
I don't use it, but it makes sense.

Reply:
> If you feel up to it, changing the loop style to `for (const auto& videoLatency : m_videoRefreshLatency)` would be nice.
Done. 
Also added the default setting as suggested. Thanks.